kvmagent/kvmagent/plugins/vm_plugin.py

```
@@ -9755,12 +9761,12 @@ def _build_domain_new_xml(self, vm, volumeDicts):
     def _build_dest_disk_xml(self, vm, oldVolumePath, newVolume):
         _, disk_name = vm._get_target_disk_by_path(oldVolumePath)
 
-        tree = etree.fromstring(vm.domain_xml)
-
-        for disk in tree.iterfind('devices/disk'):
+        root = etree.fromstring(vm.domain_xml)
```

Ruff (0.15.18)

[error] Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)